src/sentry/eventstream/base.py

```diff
@@ -172,6 +172,7 @@ def run_post_process_forwarder(
         commit_batch_timeout_ms: int,
         concurrency: int,
         initial_offset_reset: Union[Literal["latest"], Literal["earliest"]],
+        use_streaming_consumer: bool,
```

Comment:
might be good to make this named-only to avoid a [boolean trap](https://youtu.be/CnRkXO_a5mI)

Reply:
I'm just gonna leave it since it's only here for the rollout, will be probably removed next week